configure.ac

```diff
@@ -116,25 +141,11 @@ PKG_CHECK_VAR([UCX_LIBDIR],
 AS_IF([test "x$UCX_LIBDIR" = "x"],
     [AC_MSG_FAILURE([check_var succeeded, but value is incorrect])]
 )
-# Find and get info for Flux-Core using pkg-config
-AX_FLUX_CORE
-PKG_CHECK_MODULES([JANSSON],
-    [jansson >= 2.10],
-    [pkg_check_jansson_found=yes],
-    [pkg_check_jansson_found=no]
-)
-if test "x$enable_urpc" = "xyes" && test "x$pkg_check_jansson_found" = "xno"; then
-    AC_MSG_ERROR([jansson cannot be located, but it is required when '--enable-urpc' is provided])
-fi
-AM_CONDITIONAL([URPC], [test "x$enable_urpc" = "xyes"])
-AX_PERFFLOW_ASPECT([PERFFLOW],
-    [pkg_check_perfflow_found=yes],
-    [pkg_check_perfflow_found=no]
+AS_IF([test "x$with_ucx" = "xyes"],
+    [DYAD_MOD_RPATH="-Wl,-rpath,$UCX_LIBDIR"],
+    [DYAD_MOD_RPATH=""]
 )
-if test "x$enable_perfflow" = "xyes" && test "x$pkg_check_perfflow_found" = "xno"; then
```

Comment:
This got changed to with_perfflow. I would recommend only adding or removing things for UCX in this PR.

Reply:
For the time being, I just changed this to `with_perfflow`. If we want to revert that, it's a simple fix

Reply:
I am ok with it. but enable and with are different things.